sdk/module-release.json

```diff
@@ -16,9 +16,10 @@
   },
   "fileCopy": [
     {
-      "src": "../packages/checkout/widgets-lib/dist/widgets.js",
+      "src": "../packages/checkout/widgets-lib/dist/index.js",
```

Comment:
The build was outputting a file called `widgets.js` inside widgets-lib dist, but with split builds + rollup it did not seem easy/straightforward to rename the index.js file into widgets.js - so the entry file in the widgets-lib dist is just index.js but will still be renamed to widgets.js when the file ends up on jsdelivr.